tests/v2/validation/rbac/rbac_test.go

```diff
@@ -263,6 +326,50 @@ func (rb *RBTestSuite) TestRBAC() {
 	}
 }
 
+func (rb *RBTestSuite) TestRBACAdditional() {
+	rb.Run("Set up User with cluster Role for additional rbac test cases "+roleOwner, func() {
+		newUser, err := createUser(rb.client)
+		require.NoError(rb.T(), err)
+		rb.standardUser = newUser
+		rb.T().Logf("Created user: %v", rb.standardUser.Username)
+		rb.standardUserClient, err = rb.client.AsUser(newUser)
+		require.NoError(rb.T(), err)
+
+		subSession := rb.session.NewSession()
+		defer subSession.Cleanup()
+
+		rb.T().Logf("Adding user as " + roleOwner + " to the downstream cluster.")
+		//Adding created user to the downstream clusters with the role cluster Owner.
+		err = users.AddClusterRoleToUser(rb.client, rb.cluster, rb.standardUser, roleOwner)
+		require.NoError(rb.T(), err)
+		rb.standardUserClient, err = rb.standardUserClient.ReLogin()
+		require.NoError(rb.T(), err)
+
+		//Setting up an additional user for the additional rbac cases
+		additionalUser, err := createUser(rb.client)
+		require.NoError(rb.T(), err)
+		rb.additionalUser = additionalUser
+		rb.additionalUserClient, err = rb.client.AsUser(rb.additionalUser)
+		require.NoError(rb.T(), err)
+	})
+
+	rb.Run("Additional testcase1 - Validating if member with role "+roleOwner+" can add another standard user as a project owner", func() {
+		rb.ValidateAddStdUserAsProjectOwner()
+
+	})
+
+	rb.Run("Additional testcase2 - Validating if member with role "+roleOwner+" can add another standard user as a cluster owner", func() {
+		rb.ValidateAddMemberAsClusterRoles(roleOwner)
+
+	})
+
+	rb.Run("Additional testcase3 - Validating if member with role "+roleOwner+" can add a cluster member as a project owner", func() {
+		rb.ValidateAddCMAsProjectOwner(roleOwner)
```

Comment:
Depending above question, this might change

Reply:
Updated based on the feedback